src/components/elements/annotationSidePanel/components/AnnotationPolicyText.tsx

```diff
@@ -0,0 +1,135 @@
+                    className="d-flex align-items-center"
+                >
+                    <Button type="button"
+                        variant="blank"
+                        className="px-0 py-0"
+                        OnClick={() => HidePolicyText()}
+                    >
+                        <FontAwesomeIcon icon={faX}
+                            size="xl"
+                        />
+                    </Button>
+                </Col>
+                <Col className="d-flex align-items-center">
+                    <p className="fs-2 fw-lightBold">
+                        Annotation Policy
+                    </p>
+                </Col>
+            </Row>
+            {/* Policy content */}
+            <Row className="mt-5">
+                <Col>
+                    {/* User requirements */}
+                    <Row>
+                        <Col>
+                            <p className="fs-3 fw-bold">
+                                User Requirements
+                            </p>
+
+                            <ul>
+                                <li>
+                                    To create annotations, users must have a valid ORCID identifier.
+                                </li>
+                                <li>
+                                    We recommend that users add their institutional email to their ORCID record.
+                                </li>
+                            </ul>
+                        </Col>
+                    </Row>
+                    {/* Visibility of annotations */}
+                    <Row className="mt-2">
+                        <Col>
+                            <p className="fs-3 fw-bold">
+                                Visibility of Annotations
+                            </p>
+
+                            <ul>
+                                <li>
+                                    All annotations are public and can be viewed by anyone.
+                                </li>
+                                <li>
+                                    No authentication is required to view annotation records.
+                                </li>
+                                <li>
+                                    Annotations are also accessible via our API without authentication.
+                                </li>
+                            </ul>
+                        </Col>
+                    </Row>
+                    {/* Machine annotations */}
+                    <Row className="mt-2">
+                        <Col>
+                            <p className="fs-3 fw-bold">
+                                Machine Annotations
+                            </p>
+
+                            <ul>
+                                <li>
+                                    Machine agents and machine annotation services must undergo review and verification before being added to the Machine Annotation Service.
```

Comment:
before being added to the Machine Annotation Service Registry?

Reply:
yes. adding "registry" is a good idea. -> "Machine agents and machine annotation services must undergo review and verification before being added to the Machine Annotation Service Registry."